tools/test-case-generators/format-request-map.json

```diff
@@ -490,6 +490,12 @@
       "filename": "installer.iso",
       "blueprint": {
         "name": "image-installer-users",
+        "packages": [
+          {
+            "name": "podman",
```

Comment:
Why?

Reply:
1. Why not?
2. Good to have a package in there to see that it goes in the right place (payload packages), because the package set handling for the installer is a little different than other image types.  The Fedora installer in main right now ignores user/blueprint packages.